app/extensions/brave/content/scripts/flashListener.js

```diff
@@ -20,18 +22,18 @@ function isAdobeLink (href) {
       }
     })
   }
-  var observer = new window.MutationObserver(function (mutations) {
+  setTimeout(() => {
+    var observer = new window.MutationObserver(function (mutations) {
+      replaceAdobeLinks()
+    })
     replaceAdobeLinks()
```

Comment:
the check didn't really make any difference and looking at the code, that makes sense because the worst case scenario for `document.querySelector('object')` is no flash and in that case it has to scan the entire DOM just like `elem.getElementsByTagName('embed')` does. However, just adding `insertFlashPlaceholders` back in at dom load doesn't seem to effect performance in any measurable way so I'll push out that change